pkg/reconciler/broker/broker_test.go

```diff
@@ -82,6 +82,8 @@ const (
 	subscriberVersion = "v1"
 
 	brokerGeneration = 79
+
+	dependencyAnnotation = "{\"kind\":\"CronJobSource\",\"name\":\"test-cronjob-source\",\"apiVersion\":\"sources.eventing.knative.dev/v1alpha1\"}"
```

Comment:
while that thing is commented out - would it make more sense to use the newer PingSource?

Reply:
Well, it is no longer commented out :) If it's ok we can change those later, less churn in the tests?